### ISSUE TITLE
[FEATURE REQUEST] Make ObsPack default to hourly sampling when CT_sampling_strategy is not found in the data file

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#105]
## Overview
Not all ObsPack CH4 data files include the CT_sampling_strategy variable.  This can cause an error when using ObsPack.

Andy Jacobson at NOAA wrote: 

> That information is not available for our methane ObsPack. This is because the methane ObsPack is a much less mature product, and we haven't yet implemented the variable in question. You will either have to code around it, or add a CT_sampling_strategy variable to the ObsPack files. Either way, I recommend collecting either an instantaneous model sample or a simple one-hour model average to represent those measurements.

What we can do is to modify ObPack mod to:
1. First look to see if CT_sampling_strategy is found in the data file, and
2. If not, default to hourly sampling.

This is slated for GEOS-Chem 12.6.1.

**Comment by yantosca** [comment]
I have pushed commit https://github.com/geoschem/geos-chem/commit/39c3e331c7ea554f2961af8b870a428e474d2ace which now resolves this issue.  If the CT_sampling_strategy variable is not found in the ObsPack input file, we select hourly sampling, (i.e. set State_Diag%ObsPack_Strategy = 2) and then print a warning message such as this:

```
===============================================================================
GEOS-Chem WARNING: Could not find "CT_sampling_strategy" in file: 
./obspack_ch4_1_GLOBALVIEWplus_v1.0_2019-01-08.20160701.nc.  Will use hourly 
sampling by default.
 -> at ObsPack_Read_Input (in module ObsPack/obspack_mod.F90)
===============================================================================
```

This will be included in GEOS-Chem 12.6.1.

**Comment by yantosca** [comment]
Now included in GC 12.6.1, released on 28 Oct 2019.
